CRM/Civioffice/DocumentStore/Local.php

```diff
@@ -35,15 +35,24 @@ class CRM_Civioffice_DocumentStore_Local extends CRM_Civioffice_DocumentStore
     /** @var boolean should there be subfolders? */
     protected $has_subfolders;
 
-    public function __construct($uri, $name, $readonly, $has_subfolders)
-    {
-        parent::__construct($uri, $name);
-        $this->base_folder = Civi::settings()->get(self::LOCAL_STATIC_PATH_SETTINGS_KEY);
-        $this->temp_folder = Civi::settings()->get(self::LOCAL_TEMP_PATH_SETTINGS_KEY);
-        $this->readonly = $readonly;
-        $this->has_subfolders = $has_subfolders;
-    }
+  /**
+   * @phpstan-param string $uri
+   * @phpstan-param string $name
+   */
+  public function __construct($uri, $name, bool $readonly, bool $has_subfolders) {
+    parent::__construct($uri, $name);
+
+    /** @phpstan-var string|null $baseFolder */
+    $baseFolder = Civi::settings()->get(self::LOCAL_STATIC_PATH_SETTINGS_KEY);
+    $this->base_folder = is_string($baseFolder) ? trim($baseFolder) : $baseFolder;
+
+    /** @phpstan-var string|null $tempFolder */
+    $tempFolder = Civi::settings()->get(self::LOCAL_TEMP_PATH_SETTINGS_KEY);
+    $this->temp_folder = is_string($tempFolder) ? trim($tempFolder) : $tempFolder;
```

Comment:
Shouldn't it be `$this->temp_folder = is_string($tempFolder) ? trim($tempFolder) : NULL;`?